### PR TITLE
Change buffer profile threshold mode to enum

### PR DESCRIPTION
**Pull request by kcudnik**
I'n not sure but SAI_BUFFER_PROFILE_ATTR_TH_MODE should probably be CREATE_ONLY

### CODE REVIEW

**Comment by lguohan** [comment]
@kcudnik , can you update this PR?

File: inc/saibuffer.h [hunk 1]
```diff
@@ -349,9 +365,9 @@ typedef enum _sai_buffer_profile_attr_t
      * If not set, use #SAI_BUFFER_POOL_ATTR_TH_MODE.
      * (default to value set in #SAI_BUFFER_POOL_ATTR_TH_MODE)
```
Comment (lguohan) [inline inc/saibuffer.h]: remove this line as it conflicts with the below value.

Comment (kcudnik) [inline inc/saibuffer.h]: i see no conflicts ?

Comment (lguohan) [inline inc/saibuffer.h]: I mean the default value should be SAI_BUFFER_PROFILE_THRESHOLD_MODE_INHERIT_BUFFER_POOL_MODE

Comment (kcudnik) [inline inc/saibuffer.h]: but it is, line 370

Comment (kcudnik) [inline inc/saibuffer.h]: ahh, in comments, reoved

**Comment by kcudnik** [comment]
rebuild

**Pull request merged**